app/globals.css

```
@@ -0,0 +1,658 @@
+  --color-background: var(--background);
+  --color-foreground: var(--foreground);
+  --color-card: var(--card);
+  --color-card-foreground: var(--card-foreground);
+  --color-primary: var(--primary);
+  --color-primary-foreground: var(--primary-foreground);
+  --color-muted: var(--muted);
+  --color-muted-foreground: var(--muted-foreground);
+  --color-accent: var(--accent);
+  --color-border: var(--border);
+  --color-ring: var(--ring);
+  --font-sans: 'Avenir Next', 'Segoe UI', Helvetica, Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Helvetica" to be "helvetica" (value-keyword-case)

(value-keyword-case)